fix-point/LICENSE

```diff
@@ -0,0 +1,30 @@
+Copyright Author name here (c) 2018
```

Comment:
There's top-level LICENSE in project so you can remove LICENSE inside `fix-point`